src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java

```diff
@@ -109,7 +109,7 @@ public void MercuryLoginTest_009_registerButtonWorksOnJoinPage() {
     Assertion.assertEquals(registrationPage.getRegisterHeaderText(), expectedHeader);
   }
 
-  @Test(groups = "MercuryLoginTest_010")
+  @Test(groups = {"MercuryLoginTest_010", "LoginTests_002"})
   public void MercuryLoginTest_010_signInLinkWorksOnJoinPage() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)